awx/api/templates/api/mesh_visualizer.md

```diff
@@ -0,0 +1 @@
+Make a GET request to this resource to obtain a list all Receptor Nodes and their links.
```

Comment:
Are we planning to fill this out a little more? If not I would lean towards just deleting this.

Reply:
I think this was created with the understanding that this component would increase in functionality over time.  But, as that happens we can update this file....or delete it now and create this file later.